### PR TITLE
Cal 1099 availability section defined hour mismatch

### PR DESCRIPTION
**Pull request by abhisht51**
## What does this PR do?

Tries to fix the Setup Page Availability tab mismatch issue here https://github.com/calcom/cal.com/issues/7163 

Fixes # (issue)
https://github.com/calcom/cal.com/issues/7163

**Environment**: Staging

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] Chore (refactoring code, technical debt, workflow improvements)

## How should this be tested?
- Code needs additions 

## Checklist

- I haven't read the [contributing guide](https://github.com/calcom/cal.com/blob/main/CONTRIBUTING.md)
- My code doesn't follow the style guidelines of this project
- I haven't performed a self-review of my own code and corrected any misspellings
- I haven't commented my code, particularly in hard-to-understand areas
- I haven't checked if my PR needs changes to the documentation
- I haven't checked if my changes generate no new warnings
- I haven't added tests that prove my fix is effective or that my feature works
- I haven't checked if new and existing unit tests pass locally with my changes

### CODE REVIEW

**Comment by Udit-takkar** [comment]
fixed the issue 1 

TODO
- [ ] fix format of time ( it is showing 0:30 pm)

**Comment by abhisht51** [comment]
I can take this up if it's not super urgent to merge.

**Comment by Udit-takkar** [comment]
@abhisht51 👍

**Comment by abhisht51** [comment]
@Udit-takkar could you help me with the merge conflicts ?  I am not seeing them in vs code for some reason otherwise the bug is fixed

**Review by Udit-takkar — APPROVED** [review]
LGTM

File: apps/web/components/eventtype/AvailabilityTab.tsx [hunk 1]
```diff
@@ -81,6 +79,16 @@ const AvailabilitySelect = ({
       : option.value === schedules.find((schedule) => schedule.isDefault)?.id
   );
 
+  useEffect(() => {
+    if (value) {
+      formMethods.setValue("availability", value);
+    }
+  }, [value?.value]);
```
Comment (zomars) [inline apps/web/components/eventtype/AvailabilityTab.tsx:86]: Let's avoid this pattern. Let's either pass an async defaultValue to the form or don't render this until the value is defined.

**Review by zomars — CHANGES_REQUESTED** [review]

**Comment by hariombalhara** [comment]
@abhisht51 @Udit-takkar Is something pending on it?

File: packages/ui/components/navigation/tabs/VerticalTabs.tsx [hunk 2]
```diff
@@ -12,9 +12,19 @@ export interface NavTabProps {
...
+const NavTabs = function ({
+  tabs,
+  className = "",
+  sticky,
+  linkProps,
+  itemClassname,
+  iconClassName,
+  ...props
+}: NavTabProps) {
+  console.log("TABS", tabs);
```
Comment (zomars) [inline packages/ui/components/navigation/tabs/VerticalTabs.tsx:27]:
```suggestion
```

**Review by zomars — APPROVED** [review]
Fixed some conflicts, tested and is working now. 

Thank you for your contribution 🙏 

PS. tagging @leog to help testing managed events

**Pull request merged**